### PR TITLE
added Try docs in Russian

### PR DESCRIPTION
**Pull request by dobrowins**
Hi!
I wrote article in Russian about Try data type on Russian's IT site habrahabr and Raul then suggested to make a pull request with it (https://twitter.com/raulraja/status/1082265646203895808). I found article not suitable so just translated existing doc on Try in Russian. 
I am planning to translate at least all core data types this way.
Yet I wonder — how to integrate translation into the docs site? Link to translation is supposed to appear somewhere yet I am not sure about how to edit page's layout.
Now I just put translation alongside english Try's README — yet I am not sure this is correct way.

Please let me know if something else is needed to be done for this to be successfull PR!

### CODE REVIEW

**Comment by raulraja** [comment]
Awesome! can you add a link in the Try docs that points here? Thanks!

File: modules/docs/arrow-docs/docs/docs/arrow/core/try/README_RU.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+layout: docs
+title: Try
+permalink: /docs/arrow/core/try/ru
+redirect_from:
+  - /docs/datatypes/try/ru
+video: XavztYVMUqI
+---
+
+## Try
+
```
Comment (calvellido) [inline modules/docs/arrow-docs/docs/docs/arrow/core/try/README_RU.md]:
```suggestion
[English](/docs/arrow/core/try)
```

**Comment by dobrowins** [comment]
@raulraja added links to both English and Russian version of doc according to @calvellido suggestion

**Comment by raulraja** [comment]
Awesome, thanks @dobrowins !

**Pull request merged**